Documentation/ApiOverview/Ajax/Backend/Index.rst

```diff
@@ -103,7 +104,7 @@ $reasonPhrase
    {
        // our previous computation
 
-       return new Response(json_encode(['result' => $result]), 200, ['Content-Type' => 'application/json; charset=utf-8']);
+       return new JsonResponse(['result' => $result], 200);
```

Comment:
Technically we should change this to have the `ResponseFactory` injected instead and use that instead. The `JsonResponse` is internal and subject to be dropped/changed.